INSTALL.md

```diff
@@ -0,0 +1,92 @@
+### 1. Back-end setup
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/90522cc605f18974a481bd33674b4cb46d75a4ee